test/tests/postprocessors/misc_pps/misc_pps_test.i

```diff
@@ -14,16 +14,13 @@
   [./u]
     order = SECOND
     family = LAGRANGE
-    [./InitialCondition]
-      type = ConstantIC
-      value = 0
-    [../]
   [../]
 []
 
 [AuxVariables]
   [./aux_var]
     family = SCALAR
+    initial_condition = 42
```

Comment:
There's a ScalarVariable PP below that reports the value of the ScalarVariable. That's actually part of what this test is supposed to do, it shows that we can output several different Postprocessors to the Exodus file. It's actually a pretty crappy test, but that's what it does. 

The reporter was correctly reporting zero because there are no ScalarKernels in this input file but I figured that was boring. Since I had to regold it anyway, I figured I'd put a value in there so that we could "prove" that it was working properly. As you'd expect the Scalar value is correct in the initial condition and the report lags one timestep (TIMESTEP_END).